### PR TITLE
Little Fix up + Translation PL

### PR DESCRIPTION
**Pull request by deviljin112**
Fixed up other app.js missing delete Contact, moved the phone slightly more to the right of the screen, Added Polish Translation.

### CODE REVIEW

File: locales/pl.lua [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Locales['en'] = {
```
Comment (ElPumpo) [inline locales/pl.lua]: fix it

File: locales/pl.lua [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+	['new_message'] = '~b~Masz nową wiadomość:~s~ %s',
+	['press_take_call'] = '%s - Wciśnij ~INPUT_CONTEXT~ aby odebrać telefon',
+	['taken_call'] = '~y~%s~s~ odebrał telefon',
+	['gps_position'] = 'Cel podróży został dodany do GPS\'a',
+	['message_sent'] = 'Wiadomość wysłana',
+	['cannot_add_self'] = 'Nie możesz dodać samego siebie!',
```
Comment (ElPumpo) [inline locales/pl.lua]: locales start lower case (there's exceptions)

File: html/css/app.css [hunk 3]
```diff
@@ -33,7 +33,7 @@ html, body {
 	height: 480px;
 	color: #212121;
 	position: absolute;
-	right: 300px;
+	right: 50px;
```
Comment (ElPumpo) [inline html/css/app.css]: why was this changed?

File: html/scripts/app_de.js [hunk 4]
```diff
@@ -7,7 +7,10 @@
 					'<span class="sender">{{sender}}</span><br/><span class="phone-number">#{{phoneNumber}}</span>' +
 				'</div>' +
 			'</div>' +
-			'<div class="actions"><span class="new-msg newMsg-btn" data-contact-number="{{phoneNumberData}}" data-contact-name="{{senderData}}"></span></div>' +
+			'<div class="actions">' +
```
Comment (ElPumpo) [inline html/scripts/app_de.js]: why was this changed? yet again since i'm not directly going to test everything out you should've put some explaining in your PR

File: html/css/app.css (hunk 3 above)
Comment (deviljin112) [reply html/css/app.css]: As i said in my PR, moved it closer to the right as it looks slightly wrong when you pull it out whilst driving it covers the bit of the phone...

File: html/scripts/app_de.js (hunk 4 above)
Comment (deviljin112) [reply html/scripts/app_de.js]: app_de and app_en did not have the delete contact bit, i simply filled in the gap, if you look at the app.js it has the delete contact function, app_de and app_en does not... i mean just look at the code dude... its all here no one is asking you to test it :)

File: locales/pl.lua [hunk 5]
```diff
@@ -1,4 +1,4 @@
-Locales['en'] = {
+Locales['pl'] = {
 	['new_message'] = '~b~masz nową wiadomość:~s~ %s',
```
Comment (majormarcin) [inline locales/pl.lua]: @deviljin112 to może być  z dużej rozpoczęte bo jest po ~b~

**Comment by ElPumpo** [comment]
Thanks for the help!
